Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.5
+
+RUN git clone https://github.com/TeamHG-Memex/autologin.git /opt/autologin
```

Comment:
I think that line is not good because it doesn't allow to use DockerHub automated builds properly. IMHO it makes more sense to build an image from the current source code (i.e. ADD current source code), not from a fresh git clone. This way we'll be able to checkout older autologin version and build a Docker image for it using this Dockerfile; currenty Dockerfile always makes Docker build an image for autologin master.